spec/features/neighborhoods/user_searches_by_neighborhood_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+feature 'user can search by neighborhood' do
+
+  scenario 'visitor searches for a neighborhood successfully' do
+    neighborhood_1 = FactoryGirl.create(:neighborhood, name: "One Neighborhood")
```

Comment:
Useless assignment to variable - `neighborhood_1`. Did you mean `neighborhood_2`?